pkgs/by-name/an/ani-skip/package.nix

```
@@ -0,0 +1,54 @@
+{
+  stdenvNoCC,
+  fetchFromGitHub,
+  makeWrapper,
+  gnugrep,
+  gnused,
+  curl,
+  fzf,
+  lib,
+}:
+stdenvNoCC.mkDerivation (final: {
+  pname = "ani-skip";
+  version = "1.0.1";
+
+  src = fetchFromGitHub {
+    owner = "synacktraa";
+    repo = "ani-skip";
+    rev = final.version;
```

```suggestion
    rev = "refs/tags/${final.version};
```

For the sake of being explicit.